fk/main.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/fluidkeys/fluidkeys/api"
 	"github.com/fluidkeys/fluidkeys/fingerprint"
 
-	"github.com/docopt/docopt-go"
+	docopt "github.com/docopt/docopt-go"
```

Comment:
what's this change about?

Reply:
my editor wants to refence this pacakage as `docopt` when I save. I guess because the pacakge filename is `docopt-go` but I'm not sure 🤷‍♂️ Anyway, I've dropped the commit from this PR.